conduit-axum/src/server.rs

```diff
@@ -20,12 +15,8 @@ impl Server {
     /// control, such as configuring a graceful shutdown is necessary, then call
     /// `Service::from_blocking` instead.
     pub fn serve<H: conduit::Handler>(addr: &SocketAddr, handler: H) -> impl Future {
-        let handler = Arc::new(BlockingHandler::new(handler));
-        let make_service = make_service_fn(move |socket: &AddrStream| {
-            let handler = handler.clone();
-            let remote_addr = socket.remote_addr();
-            async move { Service::from_blocking(handler, remote_addr) }
-        });
+        let router = axum::Router::new().conduit_fallback(handler);
```

Comment:
nit: 

later there will probably one place where you define the central routes, which means your `Router::new` should only be in one place, right?

Reply:
Same for the middleware you apply, this also should be one place. So in tests & the normal server we have the same thing

Reply:
yeah, you're right that with the `Server` struct we won't be able to adjust the axum router. The `Server` is only used by the `conduit-axum` examples and tests though, similar to how it was with `conduit-hyper` before. Our own code is using the slightly lower level APIs directly (see `build_handler()`), where we can add axum routes and middlewares without problems :)